tests/unit/testsuite/Bolt/Boltpay/Helper/GeneralTraitTest.php

```diff
@@ -606,4 +611,52 @@ public function unserializeStringArrayProvider() {
             ),
         );
     }
+
+    /**
+     * SetUp for switch helper tests
+     */
+    private function isSwitchSetUp()
+    {
+        $this->featureSwitchMock = $this->getMockBuilder('Bolt_Boltpay_Model_FeatureSwitch')
+            ->setMethods(array('isSwitchEnabled'))->getMock();
+        Bolt_Boltpay_TestHelper::stubSingleton('boltpay/featureSwitch', $this->featureSwitchMock);
+    }
+
+    /**
+     * @test
+     * When call isSwitchSampleSwitchEnabled Bolt_Boltpay_Model_FeatureSwitch::isSwitchEnables should be called with the right parameter
+     *
+     * @covers ::isSwitchSampleSwitchEnabled
+     * @throws Exception
+     */
+    public function isSwitchSampleSwitchEnabled_shouldCallAppropriateMethodWithCorrectParameter()
+    {
+        $this->isSwitchSetUp();
+        $this->featureSwitchMock->expects($this->once())->method('isSwitchEnabled')->with('M1_SAMPLE_SWITCH');
+        $this->currentMock->isSwitchSampleSwitchEnabled();
+        $this->isSwitchTearDown();
+    }
+
+
+    /**
+     * When call isSwitchBoltEnabled Bolt_Boltpay_Model_FeatureSwitch::isSwitchEnables should be called with the right parameter
```

Comment:
@vitaliyreznikov You're missing a @test here. That's why it's not getting picked up by codecov.

Reply:
Done, thank you